config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  root 'api/greetings#index'
+  namespace :api do
+    resources :greetings, only: [:index]
+  end
```

Comment:
- [ ] When I run the two applications, I get this error in the browser console:

![image](https://github.com/DianaBeki/rails-back-end/assets/77106962/531988a3-d04f-4cbe-8426-2f56ae3f067f)

---
Your application has a Cross-Origin Resource Sharing (CORS) issue and you need to set the correct permissions for this resource to be accessible by other applications. Kindly fix this. 👍 😎 

#### Hint: To solve the issue, you can install the `rack-cors` gem and then read through their [documentation](https://github.com/cyu/rack-cors) to resolve the **_CORS policy_** issue 👍 😎